pyproject.toml

```diff
@@ -98,6 +98,38 @@ find = {namespaces = false}
 [tool.setuptools.package-data]
 dask = ["py.typed"]
 
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.lint]
+extend-select = [
+    "B",
+    "UP",
+]
```

Comment:
I think it would be better to explicitly set `select` here. That way you don't have to try and remember what is in the default rule set.